README.md

```diff
@@ -1069,9 +1069,15 @@ yet but might provide some value.
   end
   ```
 
-### Style Guides
+### Alternative Style Guides
 
-Check [Awesome Elixir][Style Guides] for a list of alternative style guides.
+* [Aleksei Magusev's Elixir Style Guide]
+  — An opinionated Elixir style guide. Developed by some of the Elixir's core
+  members and used in the Elixir project itself.
```

Comment:
please review this draft

> * [Aleksei Magusev's Elixir Style Guide](https://github.com/lexmag/elixir-style-guide#readme)
  — An opinionated Elixir style guide stemed from the coding style
  practiced in the Elixir core libraries. Developed by [Aleksei Magusev](https://github.com/lexmag) and [Andrea Leopardi](https://github.com/whatyouhide), members of Elixir's core team.
  While the Elixir project doesn't adhere to any specific style guide, this is the closest available style guide to its conventions.